libr/core/project.c

```diff
@@ -159,6 +159,56 @@ R_API int r_core_project_delete(RCore *core, const char *prjfile) {
 	return 0;
 }
 
+R_API bool r_core_rop_load(RCore *core, const char *prjfile) {
+	char *path, *db;
+	ut8 found = 0;
+	SdbListIter *it;
+	SdbNs *ns;
+
+	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
+
+	if (!prjfile || !*prjfile) {
+		return false;
+	}
+
+	if (prjfile[0] == '/') {
+		db = r_str_newf ("%s.d", prjfile);
+		if (!db) return false;
+		path = strdup (db);
+	} else {
+		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
```

Comment:
ok. by grepping a bit i found that this is exactly the way that is constructed in xrefs.c.. and this is probably the reason why some people was complaining that it was not loading the xrefs after reloading the project. I can merge this PR now, but it will be good to fix that thing in a generic way. Would you mind to update it?